apiserver/apiserver.go

```diff
@@ -985,8 +985,13 @@ func (srv *Server) endpoints() ([]apihttp.Endpoint, error) {
 		authorizer: modelCharmsUploadAuthorizer,
 	}, {
 		pattern: charmsObjectsRoutePrefix,
-		methods: []string{"GET", "PUT"},
```

Comment:
Why is PUT being removed?

Reply:
It has been moved to a handler with an authoriser